scheduler/bench/Scheduler.hs

```diff
@@ -12,30 +14,50 @@ import qualified Streamly.Prelude as S
 
 
 main :: IO ()
-main = defaultMain (mkSumBench 1000)
+main = defaultMain (mkSumBench 1000 100000)
 
-mkSumBench :: Int -> [Benchmark]
-mkSumBench n =
+mkSumBench :: Int -> Int -> [Benchmark]
+mkSumBench n elts =
   [ env (pure ls) $ \xs ->
       bgroup
         ("Sums: " <> show n)
         [ bgroup "unliftio" [bench "pooledMapConcurrently" $ nfIO (pooledMapConcurrently f xs)]
         , bgroup "monad-par" [bench "parMapM" $ nfIO (runParIO $ parMapM f xs)]
         , bgroup "scheduler" [bench "traverseConcurrently" $ nfIO (traverseConcurrently Par f xs)]
+        , bgroup "streamly" [bench "mapM" $ nfIO $ S.runStream $ asyncly $ S.mapM f $ S.fromList xs]
```

Comment:
@harendra-kumar Please send me a PR, if you think that is not the right way to traverse a stream of tasks.